src/client/client.go

```diff
@@ -34,56 +34,56 @@ func NewClient(addr string, interval int, certs TLSCerts) (*Client, error) {
 
 	c := &Client{
 		events: map[events.Event][]func(interface{}) error{
-			events.EVENT_NODE_UPDATED:               make([]func(interface{}) error, 0),
-			events.EVENT_REGISTRY_DISK_DATA_UPDATED: make([]func(interface{}) error, 0),
-			events.EVENT_VM_DATA_UPDATED:            make([]func(interface{}) error, 0),
-			events.EVENT_REGISTRY_TEMPLATES_UPDATED: make([]func(interface{}) error, 0),
+			events.EventNodeUpdated:              make([]func(interface{}) error, 0),
+			events.EventRegistryDiskDataUpdated:  make([]func(interface{}) error, 0),
+			events.EventVmDataUpdated:            make([]func(interface{}) error, 0),
+			events.EventRegistryTemplatesUpdated: make([]func(interface{}) error, 0),
 		},
 		communicator:        communicator,
 		timeoutSeconds:      int64(interval),
 		errorTimeoutSeconds: 10,
 	}
 	if err := c.communicator.TestConnection(); err != nil {
-		log.Fatal(err)
+		logger.Fatal(err)
 		return nil, err
 	}
 	return c, nil
 }
 
-func (this *Client) Init() {
+func (c *Client) Init() {
```

Comment:
There is a convention in the Go community to name receivers with single letter. Ussually, the first letter of the type name.